Mathlib/Data/Fintype/Sum.lean

```diff
@@ -4,7 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Mario Carneiro
 
 ! This file was ported from Lean 3 source module data.fintype.sum
-! leanprover-community/mathlib commit 509de852e1de55e1efa8eacfa11df0823f26f226
+! leanprover-community/mathlib commit 6623e6af705e97002a9054c1c05a980180276fc1
```

Comment:
In this file the mathlib4 version misses three `[decidableEq β]`. [#17228](https://github.com/leanprover-community/mathlib/pull/18277) says "This doesn't port well, as the bug is fixed in lean 4.", from which I assume the mathlib4-version is good as it is, is that correct?

Reply:
ok I see. Changed it now to match mathlib3.